test/suites/network_hwaddr_pattern.sh

```diff
@@ -0,0 +1,55 @@
+test_network_hwaddr_pattern() {
+    ensure_import_testimage
+    ensure_has_localhost_remote "${INCUS_ADDR}"
+
+    incus network create "inct$$"
+    incus init testimage nettest -n "inct$$"
+    incus start nettest
+
+    # First, check for the default Zabbly MAC
+    ip link show "inct$$" | grep -q 'link/ether 10:66:6a:'
+    incus exec nettest -- ip link show eth0 | grep -q 'link/ether 10:66:6a:'
+    incus stop -f nettest
+
+    # Then, override the MAC pattern globally
+    incus config set network.hwaddr_pattern=00:11:22:xx:xx:xx
+    incus config unset nettest volatile.eth0.hwaddr
+    # FIXME: ip link show "inct$$" | grep -q 'link/ether 00:11:22:'
```

Comment:
This doesn’t work because I can’t find a way to signal to the network that the server config has changed. Is it possible?
(Here, the bridge network should change its MAC address, as the parent MAC allocation pattern has changed.)